<launch>
  <node name="e4_node" pkg="empatica_e4" type="e4_serv_node.py" output="screen"/>
</launch>